frontend/src/components/DoublePanel.tsx

```diff
@@ -102,9 +104,16 @@ const useStyles = makeStyles({
   header: {
     height: spacing.lg,
   },
+  anchor: {
+    position: 'relative',
+    height: '100%',
+  },
   handle: {
+    zIndex: 5,
+    position: 'absolute',
     height: '100%',
-    padding: `${0} ${spacing.xxs}px`,
+    marginLeft: -spacing.xxs,
```

Comment:
do you have any option to avoid this negative value?

Reply:
No it's required to provide no space between panels and align the grabable border correctly.